### PR TITLE
Removal of incorrect CLA links from CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by CsatariGergely**
**What this PR does / why we need it**:

**Which issue this PR fixes** _(optional, in `fixes #<issue number>(, #<issue_number>, ...)` format, will close that issue when PR gets merged)_: fixes #

**Special notes for your reviewer**:

**Release note**:

``` release-note
```

Signed-off-by: csatari gergely.csatari@nokia.com

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/33999)

### CODE REVIEW

**Comment by CsatariGergely** [comment]
Hi I work for Nokia and Nokia signed the CLA.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -12,10 +12,8 @@ We'd love to accept your patches! Before we can take them, we have to jump a cou
 
 Please fill out either the individual or corporate Contributor License Agreement (CLA).
 
-  * If you are an individual writing original source code and you're sure you own the intellectual property, then you'll need to sign an [individual CLA](http://code.google.com/legal/individual-cla-v1.0.html).
-  * If you work for a company that wants to allow you to contribute your work, then you'll need to sign a [corporate CLA](http://code.google.com/legal/corporate-cla-v1.0.html).
-
-Follow either of the two links above to access the appropriate CLA and instructions for how to sign and return it. Once we receive it, we'll be able to accept your pull requests.
+  * If you are an individual writing original source code and you're sure you own the intellectual property, then you'll need to sign an individual CLA, which is based on the [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).
```
Comment (brendandburns) [inline CONTRIBUTING.md]: @sarahnovotny does the CNCF have their CLA posted somewhere?

Comment (sarahnovotny) [inline CONTRIBUTING.md]: @caniszczyk , is there a posted version of the CNCF CLA?

**Comment by sarahnovotny** [comment]
hi @brendandburns and @CsatariGergely ... we're getting so close.  The Google CLA is still the enforced CLA.  The CNCF CLA is running in parallel to shake out bugs in that system.  The plan is to convert to the CNCF CLA system this week if we don't find any blocking issues.  Thank you both for your patience.

**Comment by brendandburns** [comment]
@CsatariGergely you can either sign the Google CLA, or wait a little bit, it looks like.

**Comment by CsatariGergely** [comment]
Hi, okay, I wait.

**Comment by CsatariGergely** [comment]
Well, I'm just waiting for the removal of the Google CLA bot. I hope it will not take another 60 days...

**Comment by brendandburns** [comment]
@CsatariGergely needs a rebase, but Google CLA-bot is gone.

**Comment by CsatariGergely** [comment]
@brendandburns: In the meanwhile [1] was updated, and now contains the CNCF CLA liks, so this pr makes no sense anymore. 
[1]: https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md

**Comment by brendandburns** [comment]
closing due to other updates.

**Pull request closed without merging**